tests/tests38/test_http_aiohttp.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+import json
+from unittest import IsolatedAsyncioTestCase
+
+import aiohttp
+import async_timeout
+
+from mocket.mocket import Mocket, async_mocketize
+from mocket.mockhttp import Entry
+from mocket.plugins.httpretty import HTTPretty, async_httprettified, httprettified
+
+
+class AioHttpEntryTestCase(IsolatedAsyncioTestCase):
+    @async_mocketize
```

Comment:
Could you please update the `README.rst` with this full `async` testcase?

Reply:
Well, it is not possible because `IsolatedAsyncioTestCase` is available from python3.8.
What I can do is to add a disclaimer, would that be okay?

Reply:
OK, let's skip the doc for the scope of this PR. I will have a look at it later on.